tests/specs/controllers/OpenEAppController.test.js

```diff
@@ -148,4 +162,56 @@ describe('OpenEAppController', () => {
             );
         });
     });
+
+    describe('date countdown', () => {
+        beforeEach(async () => {
+            sandbox.stub(HelperService, 'getUserData').callsFake(() => ({
+                loggedIn: true,
+            }));
+            sandbox.stub(Application, 'find').resolves(resolvedAppData);
+            sandbox
+                .stub(OpenEAppController, '_getApplicationDataFromCasebook')
+                .resolves(resolvedCasebookData);
+        });
+
+        it('returns expired document error if days are negative', async () => {
+            // when
+            sandbox.stub(Date, 'now').callsFake(() => TWO_DAYS_AFTER_DEADLINE);
+            await OpenEAppController.renderPage(reqStub, resStub);
+
+            // then
+            const sailsErrorLogObj = sails.log.error.getCall(0).args[0];
+            expect(sailsErrorLogObj.message === 'Application has expired').to.be
+                .true;
+            expect(resStub.serverError.called).to.be.true;
+        });
+        it('shows correct number of days for 11 day old application', async () => {
+            // when
+            sandbox.stub(Date, 'now').callsFake(() => ELEVEN_DAYS_BEFORE_DEADLINE);
+            await OpenEAppController.renderPage(reqStub, resStub);
+
+            // then
+            expectedPageData.daysLeftToDownload = 10;
```

Comment:
should this be an `expect`? it is just being assigned as 10 here. is that right?

Reply:
Not quite, so I'm passing in a date 11 days before the deadline and expecting the page data to return 10 days. If the 10 in this line was changed the tests would fail, well should fail.

Re-reading that I'm not sure if it makes sense 🤔